parsing.js

```diff
@@ -107,13 +116,16 @@ function fetchVerse(verse_id) {
  *  @param {*} document
  *  @returns an array of key value pairs (chapter:verse): index
  */
+
+// ex: 85:1
 function getVerseIndices(document) {
     let new_format;
     let content = document.body.content;
     let verses = {};
     let verse;
     content.forEach((line, line_index) => {
         line?.paragraph?.elements.forEach((element) => {
+            // what is the horizontal rule??
```

Comment:
Discussed in the meeting. It's a horizontal line seen in the original test document. Leftover comment.